specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/DynamicSessions.json

```diff
@@ -770,6 +1074,23 @@
       "required": [
         "name"
       ]
+    },
+    "SessionsListResponse": {
+      "type": "object",
+      "description": "The sessions list response.",
+      "properties": {
+        "sessions": {
+          "type": "array",
+          "description": "The list of sessions.",
+          "items": {
```

Comment:
Just a suggestion, ignore if not relevant: Do we want to send "nextLink" field in the response body with an absolute URL that the client can GET in order to retrieve the next page of the collection.

https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options:~:text=DO%20return%20a%20nextLink%20field%20with

Reply:
done